contentpacks/utils.py

```diff
@@ -704,3 +704,14 @@ def remove_nonexistent_assessment_items_from_exercises(node_data: list, assessme
             except Exception as e:
                 import pdb; pdb.set_trace()
                 print(1)
+
+def remove_exercises_without_assessment_items(node_data):
```

Comment:
It would be good to merge the functionality of the `remove_nonexistent_assessment_items_from_exercises()` above with your `remove_exercises_without_assessment_items()` function so that it saves processing time and works the same way.  For now let's make this fix work.